chains/evm/scripts/compile_all

```diff
@@ -23,23 +23,29 @@ compileContract() {
   contract=$(basename "$1")
   echo "Compiling" "$contract"
   dir=$CONTRACTS_DIR/solc/$PROJECT/$contract
-
-  local command
-  command="forge build $CONTRACTS_DIR/contracts/"$1.sol" \
-   --root $CONTRACTS_DIR \
-   $(getOptimizations "$contract") \
-   --extra-output-files bin abi metadata \
-   --build-info \
-   --build-info-path $dir/build \
-   -o $dir"
-
-  $command
-
-  # Move the build info to an expected file name
-  mv $(find $dir/build -type f -name '*.json' ! -name 'build.json') $dir/build/build.json
-
-  # Copy the generated abi files to a single folder
-  cp "$CONTRACTS_DIR"/solc/$PROJECT/"$contract"/"$contract".sol/"$contract".abi.json "$ABI_DIR""$contract".abi
+  
+  local args # base args for forge
+  args="build $CONTRACTS_DIR/contracts/"$1.sol" --root $CONTRACTS_DIR"
+  
+  if [ "$ZKSYNC" != "true" ]; then
```

Comment:
same as above, please consider swapping the order to make the happy path for zksync more explicit

Reply:
Same. Fixing it